README.rst

```diff
@@ -15,7 +15,7 @@ Overview
       - |version| |downloads| |wheel| |supported-versions| |supported-implementations|
 
 .. |docs| image:: https://readthedocs.org/projects/pytest-cov/badge/?style=flat
-    :target: https://readthedocs.org/projects/pytest-cov
+    :target: https://pytest-cov.readthedocs.io/
```

Comment:
This link does not need to change. It's rather a link for me to check docs build status, not for users.

For users we should have `Documentation` section (I think above `Installing`) in the `README.rst`. We don't have it yet, oops. Can you add it and revert this part?